xiaomi_gateway/__init__.py

```diff
@@ -194,7 +191,7 @@ class XiaomiGateway:
     """Xiaomi Gateway Component"""
 
     # pylint: disable=too-many-arguments
-    def __init__(self, ip_adress, port, sid, key, discovery_retries, interface, proto=None):
+    def __init__(self, ip_adress, port=MULTICAST_PORT, sid, key, discovery_retries, interface, proto=None):
```

Comment:
You can't do named arguments before unnamed arguments, so this has to become something like:
```
    def __init__(self, ip_adress, sid, key, discovery_retries, interface, port=MULTICAST_PORT, proto=None):
        ....
```